Sources_v3/Query/ChannelQuery.swift

```diff
@@ -28,9 +28,10 @@ public struct ChannelQuery<ExtraData: ExtraDataTypes>: Encodable {
     /// ChannelCreatePayload that is needed only when creating channel
     let channelPayload: ChannelEditDetailPayload<ExtraData>?
     
-    /// ChannelId this query handles.
-    var cid: ChannelId {
-        guard let id = id else { return .init(type: type, id: "") }
+    /// `ChannelId` this query handles.
+    /// If `id` part is missing then it's impossible to create valid `ChannelId`.
+    public var cid: ChannelId? {
+        guard let id = id else { return nil }
         return ChannelId(type: type, id: id)
```

Comment:
nit: You can do something like `return id.map { ChannelId(type: type, id: $0) }`